internal/agent/agent.go

```diff
@@ -422,9 +428,37 @@ func New(
 
 	// Create input channel for plugins to feed data back to the agent
 	llog.WithField(config.TracesFieldName, config.FeatureTrace).Tracef("inventory parallelize queue: %v", a.Context.cfg.InventoryQueueLen)
-	a.Context.ch = make(chan PluginOutput, a.Context.cfg.InventoryQueueLen)
+	a.Context.ch = make(chan types.PluginOutput, a.Context.cfg.InventoryQueueLen)
 	a.Context.activeEntities = make(chan string, activeEntitiesBufferLength)
 
+	if cfg.InventorySendBulk {
+		removeEntitiesPeriod, _ := time.ParseDuration(a.Context.Config().RemoveEntitiesPeriod)
```

Comment:
Should we use `defaultRemoveEntitiesPeriod` in case of error?

Reply:
it is, but in the patch sender (we will remove inventory logic from agent.go)